run_bot.sh

```
@@ -1,1082 +1,194 @@
-        echo "./recordings/$bot_uuid/output.mp4"
-        echo "./recordings/$bot_uuid/"  # folder for metadata and all files
-    fi
-    
-    print_info "Full log available at: $log_file"
-    
-    # Cleanup
-    rm -f "$log_file"
+    [ ! -d "$output_dir" ] && { print_info "No recordings directory"; return; }
+    read -p "Delete all recordings? (y/N): " -n 1 -r
+    echo
+    [[ $REPLY =~ ^[Yy]$ ]] && { rm -rf "$output_dir"/*; print_success "Cleaned"; } || print_info "Cancelled"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)